README.md

```diff
@@ -1,35 +1,42 @@
-# OpenPower Firmware Build Environment
+# OpenPOWER Firmware Build Environment
```

Comment:
I just noticed the foundation decided to use this case. For years IBM used POWER and only recently changed to Power. I'm surprised we went back to OpenPOWER.